src/github.com/getlantern/elevate/README.md

```diff
@@ -0,0 +1,6 @@
+[Godoc](http://godoc.org/github.com/getlantern/elevate)
```

Comment:
It would be better to add an example on each OS in the future. It's ok for this PR.